<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFileDialog</class>
 <widget class="QDialog" name="AddFileDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增文件</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_addselect">
        <property name="text">
         <string>添加选中</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_newlabel">
        <property name="text">
         <string>新的标签</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_removelabel">
        <property name="text">
         <string>移除标签</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="lv_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
